caikit/core/toolkit/errors/error_handler.py

```diff
@@ -273,7 +273,7 @@ def type_check_all(self, log_code, *types, allow_none=False, **variables):
                         ),
                     )
 
-    def value_check(self, log_code, condition, *args, msg=""):
+    def value_check(self, log_code, condition, msg, *args):
```

Comment:
I couldn't image a scenario in which msg is an empty string so removed the default value for it.

Reply:
I think the idea was if you just didn't want a message, you didn't need one